src/mcp_server_helm/commands/basic.py

```
@@ -0,0 +1,64 @@
+    valid_shells = ["bash", "fish", "powershell", "zsh"]
+    if shell not in valid_shells:
+        return f"Invalid shell: {shell}. Valid options are: {', '.join(valid_shells)}"
+
+    return execute_helm_command(["helm", "completion", shell])
+
+
+def helm_create(name: str, starter: Optional[str] = None) -> str:
+    """
+    Creates a new chart with the given name.
+    """
+    logger.info(f"Running helm create with name={name}, starter={starter}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)